setup.cfg

```diff
@@ -79,10 +79,10 @@ runtime =
     opensearch-py==1.1.0
     pproxy>=2.7.0
     pyopenssl>=21.0.0
-    Quart>=0.6.15
+    Quart==0.17
```

Comment:
@alexrashed not sure if this is actually an issue but this should be `quart` and not `Quart`

Reply:
Interesting, TIL that PIP / PEP-426 is case-insensitive:
https://peps.python.org/pep-0426/#name
> All comparisons of distribution names MUST be case insensitive, and MUST consider hyphens and underscores to be equivalent.